src/components/User/Sales/components/SaleList.vue

```diff
@@ -223,23 +218,27 @@ export default {
 
   watch: {
     'filters.openOnly' () {
-      this.getList()
+      this.$refs.collectionLoaderBtn.loadFirstPage()
     },
     'owner': _.throttle(function () {
       this.getOwner()
-      this.getList()
+      this.$refs.collectionLoaderBtn.loadFirstPage()
     }, 1000),
     'filters.name': _.throttle(function () {
-      this.getList()
+      this.$refs.collectionLoaderBtn.loadFirstPage()
     }, 1000),
     'filters.baseAsset': _.throttle(function () {
-      this.getList()
+      this.$refs.collectionLoaderBtn.loadFirstPage()
     }, 1000),
     'filtersDate.startDate' () {
       this.filterByDate()
+      // hides collection-loader button
+      this.$refs.collectionLoaderBtn.isCollectionFetched = true
     },
     'filtersDate.endDate' () {
       this.filterByDate()
+      // hides collection-loader button
+      this.$refs.collectionLoaderBtn.isCollectionFetched = true
```

Comment:
oof, the parent should not be responsible for the children states. maximum the parent can is execute some methods, like you do with `this.$refs.collectionLoaderBtn.loadFirstPage()`